app/seksjon/verneplikt/v1/oppsummering/VernepliktOppsummeringV1.tsx

```diff
@@ -3,10 +3,10 @@ import { FormSummary } from "@navikt/ds-react";
 import OppsummeringsSvar from "~/components/OppsummeringsSvar";
 import { SeksjonProps } from "~/utils/oppsummering.utils";
 
-export default function VernepliktOppsummering({ seksjonsData, seksjonsUrl }: SeksjonProps) {
-  if (!seksjonsData) return <></>;
+export default function VernepliktOppsummeringV1({ seksjonSvarene, seksjonsUrl }: SeksjonProps) {
+  if (!seksjonSvarene) return <></>;
 
-  const vernepliktData: VernepliktSvar = JSON.parse(seksjonsData);
+  const vernepliktData: VernepliktSvar = seksjonSvarene as VernepliktSvar;
```

Comment:
Den her blir litt redundant med å spesifisere type to ganger. Holder med `const vernepliktData = seksjonSvarene as VernepliktSvar;`